cmd/utils/history_test.go

```diff
@@ -113,6 +113,7 @@ func TestHistoryImportAndExport(t *testing.T) {
 			if err != nil {
 				t.Fatalf("error opening era file: %v", err)
 			}
+			defer f.Close()
```

Comment:
This is a bit non-obvious,but `era.From` takes an `io.ReadSeekCloser`, and `era.Close` closes the `f`. So this change would make it a double-close